stdlib/3/http/__init__.pyi

```diff
@@ -10,60 +10,60 @@ if sys.version_info >= (3, 5):
         phrase = ...  # type: str
```

Comment:
These will probably also be treated strangely by the type system; can you annotate them as `@property`?

Reply:
I don't mind updating `phrase` and `description` as `@property`s, though I should point out that they're not defined that way in the stdlib. They're just instance variables populated during `__new__`. There are no explicit getters for either.

Whatever you think is best; I'm new to mypy and typeshed.

FWIW, haven't (yet) had any type checking problems with either `phase` or `description`.

Reply:
OK, let's leave this for now.